src/board.py

```diff
@@ -1,13 +1,14 @@
+from dataclasses import dataclass
 from typing import List
 
 from piece import Piece
 
 
 class Board:
-    def __init__(self, number_players: int = 4, numberOfPieces: int = 4):
+    def __init__(self, number_players: int = 4, number_of_pieces: int = 4):
         self._pieces: List[Piece] = []
         for player in range(1, number_players + 1):
```

Comment:
There is some arithmetic in the `draw_board` implementation that expects `0` indexed players.

Reply:
I am also for 0 indexed counting.
This  line hasn't changed in this PR.